parade/ddl.sql

```diff
@@ -127,6 +131,7 @@ WITH signalled_ids AS (
     SET total_dependencies = tasks.total_dependencies-1
     WHERE tasks.id IN (SELECT UNNEST(to_signal_after) FROM tasks WHERE id=task_id)
     RETURNING (CASE WHEN tasks.total_dependencies = 0 THEN tasks.id ELSE NULL END) id
+    -- tzahi: shouldn't you be comparing the total_dependencies to num_signals ? the tasks may have already received signals
```

Comment:
Yes, I should!  It's a bug.  `delete_task` is mostly useful in testing, but it still needs to be correct :-)  PR forthcoming.